dep/src/main/java/badClassName.java

```diff
@@ -0,0 +1,19 @@
+
+
+public class badClassName {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+
+	}
+	
+	public void BadMethodName(){
+		System.out.println("asdfsa");
+		return;
+	}
+
+	public void BadMethodName2(){
+		System.out.println("asdfsa");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)